docs/tutorial/kymographs.rst

```diff
@@ -5,37 +5,65 @@ Kymographs
 
     :nbexport:`Download this page as a Jupyter notebook <self>`
 
-To load an HDF5 file and lists all of the kymographs inside of it, run::
+We can use the :attr:`lk.File.kymos <lumicks.pylake.File.kymos>` attribute to access the kymographs from a file::
 
     import lumicks.pylake as lk
 
-    file = lk.File("example.h5")
-    list(file.kymos)  # e.g. shows: "['cas9', 'reference']"
+    file = lk.File("kymo.h5")
+    print(file.kymos)  # dict of available kymos: {'16': Kymo(pixels=699)}
 
-Once again, :attr:`.kymos <lumicks.pylake.File.kymos>` is a regular Python dictionary so we can
-easily iterate over it::
+This is a regular Python dictionary so we can easily iterate over it::
 
-    # Plot all kymos in a file
-    >>> for name, kymo in file.kymos.items():
-            print(f"{name}, starts at {kymo.start} ns")
-    cas9, starts at 1586776312560250200 ns
-    reference, starts at 1586777007674285402 ns
+    # print some details for all kymos in a file
+    for name, kymo in file.kymos.items():
+        print(f"kymograph '{name}', starts at {kymo.start} ns")
 
-Or just pick a single one::
+    # kymograph '16', starts at 1638534513847557200 ns
 
-    kymo = file.kymos["cas9"]
-    kymo.plot(channel="green", aspect="auto", adjustment=lk.ColorAdjustment(0, 5))
+Or access a particular `Kymo` object directly to work with::
+
+    # use the kymo name as a dict key
+    kymo = file.kymos["16"]
+
+    # or get the first kymo from the dict
+    kymo_name, kymo = file.kymos.popitem()
```

Comment:
I have mixed feelings about adding `.popitem()` here because it can lead to confusion unless you actually explain what it does exactly. Reason being that if they'd do this:

```python
kymos = file.kymos
kymo_name, kymo = kymos.popitem()
```

 It would remove the `kymo` from the dictionary.

Reply:
Ah I was wondering why it wasn't removed. This is what I always use because then I don't have to look up the kymo name first, so I thought it might be nice to add here; but I get your point. Might be nice to add some kind of functionality to get the kymos by index rather than by name, but that's another ticket